src/backend/function/Fusion.Resources.Functions/ApiClients/LineOrgApiClient.cs

```diff
@@ -0,0 +1,43 @@
+﻿#nullable enable
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Fusion.Resources.Functions.Integration;
+
+namespace Fusion.Resources.Functions.ApiClients;
+
+public class LineOrgApiClient : ILineOrgApiClient
+{
+    private readonly HttpClient lineOrgClient;
+
+    public LineOrgApiClient(IHttpClientFactory httpClientFactory)
+    {
+        lineOrgClient = httpClientFactory.CreateClient(HttpClientNames.Application.LineOrg);
+    }
+
+    public async Task<IEnumerable<string>> GetOrgUnitDepartmentsAsync()
+    {
+        var data =
+            await lineOrgClient.GetAsJsonAsync<InternalCollection<DepartmentRef>>($"/org-units?$top={int.MaxValue}");
```

Comment:
I guess we have  mayebe 1000 as default in the API? Seems a little hacky with int.maxvalue, but I guess that will work.